pulp_smash/tests/pulp3/file/api_v3/test_crud_importers.py

```diff
@@ -0,0 +1,99 @@
+# coding=utf-8
+"""Tests that CRUD importers."""
+import unittest
+
+from requests.exceptions import HTTPError
+
+from pulp_smash import api, config, selectors
+from pulp_smash.tests.pulp3.constants import FILE_IMPORTER_PATH, REPO_PATH
+from pulp_smash.tests.pulp3.file.api_v3.utils import (
+    gen_importer,
+    modify_importer_down_policy,
+    modify_importer_sync_mode,
+)
+from pulp_smash.tests.pulp3.pulpcore.utils import gen_repo
+from pulp_smash.tests.pulp3.utils import get_auth
+
+
+class CRUDImportersTestCase(unittest.TestCase):
+    """CRUD importers."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create class-wide variables.
+
+        In order to create an importer a repository has to be created first.
+        """
+        cls.cfg = config.get_config()
+        cls.client = api.Client(cls.cfg, api.json_handler)
+        cls.client.request_kwargs['auth'] = get_auth()
+        cls.importer = {}
+        cls.repo = cls.client.post(REPO_PATH, gen_repo())
+
+    @classmethod
+    def tearDownClass(cls):
+        """Clean class-wide variable."""
+        cls.client.delete(cls.repo['_href'])
+
+    @selectors.skip_if(bool, 'repo', False)
```

Comment:
This selector can be dropped. If `cls.client.post(REPO_PATH, gen_repo())` returns an error, then `setUpClass` will not successfully complete, and none of the `test*` methods will run.